samples/scala-customer-registry-quickstart/build.sbt

```diff
@@ -0,0 +1,38 @@
+name := "customer-registry"
+
+organization := "customer"
+organizationHomepage := Some(url("https://akkaserverless.com"))
+licenses := Seq(("CC0", url("https://creativecommons.org/publicdomain/zero/1.0")))
+
+scalaVersion := "2.13.6"
+
+enablePlugins(AkkaserverlessPlugin, JavaAppPackaging, DockerPlugin)
+dockerBaseImage := "docker.io/library/adoptopenjdk:11-jre-hotspot"
+dockerUsername := sys.props.get("docker.username")
+dockerRepository := sys.props.get("docker.registry")
+ThisBuild / dynverSeparator := "-"
+
+Compile / scalacOptions ++= Seq(
+  "-target:11",
+  "-deprecation",
+  "-feature",
+  "-unchecked",
+  "-Xlog-reflective-calls",
+  "-Xlint")
+Compile / javacOptions ++= Seq("-Xlint:unchecked", "-Xlint:deprecation", "-parameters" // for Jackson
+)
+
+Test / parallelExecution := false
+Test / testOptions += Tests.Argument("-oDF")
+Test / logBuffered := false
+
+Compile / run := {
+  // needed for the proxy to access the user function on all platforms
+  sys.props += "akkaserverless.user-function-interface" -> "0.0.0.0"
+  (Compile / run).evaluated
+}
```

Comment:
@raboof would this be something the sbt-plugin could set, to simplify the user's build?